ScoutSuite/providers/aws/services/cloudtrail.py

```diff
@@ -45,17 +45,23 @@ def parse_trail(self, global_params, region, trail):
                 if key not in trail_config:
                     trail_config[key] = False
             trail_details = api_client.get_trail_status(Name=trail['TrailARN'])
-            for key in ['IsLogging', 'LatestDeliveryTime', 'LatestDeliveryError', 'StartLoggingTime', 'StopLoggingTime', 'LatestNotificationTime', 'LatestNotificationError', 'LatestCloudWatchLogsDeliveryError', 'LatestCloudWatchLogsDeliveryTime']:
+            for key in ['KMSKeyId', 'IsLogging', 'LatestDeliveryTime', 'LatestDeliveryError', 'StartLoggingTime', 'StopLoggingTime', 'LatestNotificationTime', 'LatestNotificationError', 'LatestCloudWatchLogsDeliveryError', 'LatestCloudWatchLogsDeliveryTime']:
                 trail_config[key] = trail_details[key] if key in trail_details else None
+                
 
         if trail_details:
             # using trail ARN instead of name as with Organizations the trail would be located in another account
             trail_config['wildcard_data_logging'] = self.data_logging_status(trail_config['TrailARN'],
                                                                              trail_details,
                                                                              api_client)
-
+            
+            for es in api_client.get_event_selectors(TrailName=trail_config['TrailARN'])['EventSelectors']:
+                trail_config['DataEventsEnabled'] = True if len(es['DataResources']) > 0 else False
```

Comment:
I think we can simplify this line.
```suggestion
                trail_config['DataEventsEnabled'] = len(es['DataResources']) > 0
```